3rdparty/tiny/api/internally_implemented.cpp

```diff
@@ -0,0 +1,325 @@
+/*
```

Comment:
would it be clearer to name the directory `3rdparty/mlperftiny/api/`?

Reply:
I matched it with their repo name, but I agree with you. I'll also add a readme to explain where did we get these files.